mongodbatlas/resource_mongodbatlas_custom_db_role.go

```diff
@@ -0,0 +1,301 @@
+package mongodbatlas
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"regexp"
+	"strings"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	matlas "github.com/mongodb/go-client-mongodb-atlas/mongodbatlas"
+	"github.com/mwielbut/pointy"
+	"github.com/spf13/cast"
+)
+
+func resourceMongoDBAtlasCustomDBRole() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceMongoDBAtlasCustomDBRoleCreate,
+		Read:   resourceMongoDBAtlasCustomDBRoleRead,
+		Update: resourceMongoDBAtlasCustomDBRoleUpdate,
+		Delete: resourceMongoDBAtlasCustomDBRoleDelete,
+		Importer: &schema.ResourceImporter{
+			State: resourceMongoDBAtlasCustomDBRoleImportState,
+		},
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"role_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: validation.All(
+					validation.StringMatch(regexp.MustCompile(`[\w\d-]+`), "`role_name` can contain only letters, digits, underscores, and dashes"),
+					func(v interface{}, k string) (ws []string, es []error) {
+						value := v.(string)
+						log.Printf("LOG___ value: %#+v\n", value)
```

Comment:
Could you please remove this log?

Reply:
Nice! thank you so much :) let me remove it 👍